core/src/main/scala/com/rallyhealth/vapors/core/dsl/ExprDsl.scala

```diff
@@ -77,6 +77,9 @@ trait ExprDsl extends WrapExprSyntax with WrapEachExprSyntax {
   ): Expr.Embed[V, R, P] =
     Expr.Embed(expr, captureResult)
 
+  def concat[V, M[+_], R, P](expressions: Expr[V, M[R], P]*): ConcatOutputExprBuilder[V, M, R, P] =
```

Comment:
BLKR: I'm not sure if this restriction on M[] to be covariant is necessary. If not, I should remove it.